specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/subscriptions.json

```diff
@@ -156,6 +156,70 @@
           }
         ]
       }
+    },
+    "/providers/Microsoft.Subscription/subscriptions/{subscriptionId}/cancel": {
+      "post": {
+        "operationId": "CancelSubscription",
+        "description": "Cancels the subscription",
+        "parameters": [
+          {
+            "name": "subscriptionId",
```

Comment:
Parameter "subscriptionId" is not allowed in the operations section, define it in the global parameters section instead

Reply:
Moved to parameters